packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1879,6 +1938,37 @@ export class TransactionController extends BaseController<
       this.updateTransaction.bind(this),
     );
   }
+
+  private async signTransaction(
+    transactionMeta: TransactionMeta,
+  ): Promise<string | undefined> {
+    const { txParams } = transactionMeta;
+
+    const unsignedEthTx = this.prepareUnsignedEthTx(txParams);
+    const signedTx = await this.sign?.(
+      unsignedEthTx,
+      txParams.from,
+      ...this.getAdditionalSignArguments(transactionMeta),
+    );
+    if (!signedTx || !this.afterSign(transactionMeta, signedTx)) {
+      return undefined;
```

Comment:
Can we take advantage of the `log` object and track this with something like `Skipping signed status based on hook`?